src/BonsaiBalance.Mobile/BaseProject/.watchmanconfig

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
?

Reply:
that should probably be in the gitignore